dubbo-plugin/dubbo-reactive/src/main/java/org/apache/dubbo/reactive/ServerTripleReactorSubscriber.java

```diff
@@ -20,11 +20,24 @@
 import org.apache.dubbo.rpc.protocol.tri.CancelableStreamObserver;
 import org.apache.dubbo.rpc.protocol.tri.observer.CallStreamObserver;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
 /**
  * The Subscriber in server to passing the data produced by user publisher to responseStream.
  */
 public class ServerTripleReactorSubscriber<T> extends AbstractTripleReactorSubscriber<T> {
 
+    private final List<T> collectedData = new ArrayList<>();
+    private final CompletableFuture<List<T>> completableFuture = new CompletableFuture<>();
```

Comment:
1. Make naming `completableFuture` more meaningful.
2. Why is List<T> type? Is combined future?